src/Components/Form/TextInput/TextInput.jsx

```diff
@@ -31,13 +36,34 @@ const TextInput = ({
     setFlow(!!value);
   }, [value]);
 
+  const onInputChange = useCallback(
+    (input) => {
+      if (type !== "number") {
+        return onChange(input);
+      } else {
+        if (input > max) {
+          return onChange(max);
+        } else if (input < min) {
```

Comment:
I think this feals a little bit buggy. But there is no easy solution for now. The best for a long time should be to use a form state lib (like I already proposed.) and only show an error and disable the submit button.

Reply:
I think we don't need the if dimensions due to early return.